targets/ath79-generic

```diff
@@ -86,6 +86,11 @@ device('buffalo-wzr-hp-g300nh-rtl8366s', 'buffalo_wzr-hp-g300nh-s')
 
 -- devolo
 
+device('devolo-dlan-pro-1200+-wifi-ac', 'devolo_dlan-pro-1200plus-ac', {
+	packages = ATH10K_PACKAGES_QCA9880,
+	factory = false,
+})
+
```

Comment:
As the LED is not working, please add broken here.

Reply:
There is no Power LED which should work that way in the first place.
1. house LED for Powerline (DLAN)
2. wifi LED for wireless activity

I am happy adding this as broken.